tests/modules/asset_groups/test_tus_creation.py

```diff
@@ -16,6 +18,9 @@ def test_create_submission_from_tus(db, researcher_1, test_root):
     tid = get_transaction_id()  # '11111111-1111-1111-1111-111111111111'
 
     # first no such dir exists
+    transaction_dir = pathlib.Path(tus_upload_dir(flask_app, transaction_id=tid))
+    if (transaction_dir).exists():
```

Comment:
i think this is a totally acceptable solution for this fix :+1: 
but i wonder -- does this imply some _other_ test is being sloppy and leaving one behind?  seems to be the case.
not really urgent -- but just kind of noting it.

Reply:
Yeah, I think other tests that are also using `get_transaction_id()` isn't cleaning up...